src/profile/components/Statistics/Orders/orders.less

```diff
@@ -15,6 +15,7 @@
 }
 
 .centerChart {
+  height: 550px;
```

Comment:
Would this not force the size of all the charts, kinda breaking the card for the pie chart?

Reply:
You're absolutely correct, didn't notice it because it only looked *slightly* broken :innocent: